src/components/Nav.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import whiteLogo from '../images/white-logo-tinder.png';
+import colorLogo from '../images/color-logo-tinder.png';
+
+
+function Nav({minimal, authToken}) {
+
+    // const minimal = true;
```

Comment:
Nit: Could you please remove this variable since you are not currently using it?